met-web/src/components/survey/edit/EditForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react';
+import { Grid, Stack, TextField } from '@mui/material';
+import { ActionContext } from './ActionContext';
+import { MetLabel, PrimaryButton, SecondaryButton } from 'components/common';
+import { SurveyFormProps } from '../types';
+
+export const EditForm = ({ handleClose }: SurveyFormProps) => {
+    const { handleSubmit, isSubmitting, submission } = useContext(ActionContext);
+
+    return (
+        <Grid
+            container
+            direction="row"
+            justifyContent="flex-start"
+            alignItems="flex-start"
+            spacing={2}
+            padding={'2em 2em 1em 2em'}
+        >
+            {submission?.comments.map((comment, index) => {
+                return (
+                    <Grid item xs={12} key={index}>
+                        <MetLabel>{comment.label}</MetLabel>
+                        <TextField
+                            defaultValue={comment.text}
+                            sx={{ width: '100%' }}
+                            multiline={true}
+                            rows={3}
+                            onChange={(e) => {
+                                comment.text = e.target.value;
```

Comment:
I feel weird about handling state change like this. I think it could be against react pattern because state change should always occur using setState hook or this.setState for class components. Maybe we could pass the setSubmission and use the index to update the particular comment?